backend/danswer/connectors/slack/connector.py

```diff
@@ -328,7 +316,44 @@ def get_all_docs(
         )
 
 
```

Comment:
Added this to make it an order of magnitude faster than running the whole connector each  time